pyproject.toml

```diff
@@ -140,8 +142,38 @@ flake8 = "3.8.4"
 black = "^22.6.0"
 # Latest version compatible with `trio_typing~=0.5`, required by qtrio
 mypy = "v0.910"
+
 setuptools = "^63.1.0"
+maturin = "~=0.13"
+cibuildwheel = "2.8.1"
+
+
+[tool.poetry.build]
+generate-setup-file = false
+script = "build.py"
+
+[tool.cibuildwheel]
+build = "cp39-{manylinux,macos,win}*"
+archs = ["auto64"]
+build-verbosity = 3
+before-all = "bash ci/setup-rust.sh"
+before-build = "python -m pip install -U pip"
+
+[tool.cibuildwheel.environment]
+PATH = "$PATH:$HOME/.cargo/bin"
```

Comment:
done

Reply:
The comment should explain *why* it we need to have this folder among the PATH

I guess this is needed because rust might not be present on the system (given `setup-rust.sh` detect and install it if needed). In this case, `$HOME/.cargo/bin` doesn't exist when cibuildwheel is started, hence the PATH environment is missing it (and `setup-rust.sh` cannot modify an environment variable for it parents process), henc why we have to force the path by hand.

Reply:
Did you read the comment that I've added ? If the comment need change, open a new discussion at the start of the comment